### PR TITLE
routing: (almost) support street network when starting to a stop_point

### PR DESCRIPTION
**Pull request by TeXitoi**
This PR makes `max_duration_to_pt` completely forbid using the street network.

Everything is here to activate the street network at the stop points, just remove the 2 TODO lines and update the test by uncommenting them.

### CODE REVIEW

File: source/jormungandr/tests/ptref_tests.py [hunk 1]
```diff
@@ -755,7 +755,7 @@ def test_headsign_stop_time_vj(self):
 
     def test_headsign_display_info_journeys(self):
         """test basic print of headsign in section for journeys"""
-        response = self.query_region('journeys?from=stop_point:stopB&to=stop_point:stopA&datetime=20120615T000000')
+        response = self.query_region('journeys?from=stop_point:stopB&to=stop_point:stopA&datetime=20120615T000000&max_duration_to_pt=0')
```
Comment (xlqian) [inline source/jormungandr/tests/ptref_tests.py:758]: :+1:

File: source/routing/routing.cpp [hunk 2]
```diff
@@ -64,10 +64,13 @@ std::string PathItem::print() const {
     return ss.str();
 }
 
+bool is_same_stop_point(const type::EntryPoint& point, const type::StopPoint* stop_point) {
```
Comment (xlqian) [inline source/routing/routing.cpp]: static bool ...

Comment (TeXitoi) [reply source/routing/routing.cpp]: nop, exported and used in raptor_api

File: source/routing/raptor_api.cpp [hunk 3]
```diff
@@ -554,7 +554,9 @@ static void add_pathes(PbCreator& pb_creator,
             const auto& departure_stop_point = path.items.front().stop_points.front();
             georef::Path sn_departure_path = worker.get_path(departure_stop_point->idx);
 
-            if (use_crow_fly(origin, departure_stop_point, sn_departure_path, pb_creator.data)){
+            if (is_same_stop_point(origin, departure_stop_point)) {
+                // nothing in this case
+            } else if (use_crow_fly(origin, departure_stop_point, sn_departure_path, pb_creator.data)){
```
Comment (xlqian) [inline source/routing/raptor_api.cpp]: How about?

``` c++
if (! is_same_stop_point(origin, departure_stop_point) && use_crow_fly(origin, departure_stop_point, sn_departure_path, pb_creator.data))
```

File: source/routing/routing.cpp (hunk 2 above)
Comment (xlqian) [reply source/routing/routing.cpp]: ok

File: source/routing/routing.cpp [hunk 4]
```diff
@@ -64,10 +64,13 @@ std::string PathItem::print() const {
...
+    return point.type == type::Type_e::StopPoint && point.uri == stop_point->uri;
```
Comment (xlqian) [inline source/routing/routing.cpp]: just a detail, if stop_point can't be null, I prefer use a ref, if stop_point can be null, it should be checked :)

File: source/routing/raptor_api.cpp (hunk 3 above)
Comment (TeXitoi) [reply source/routing/raptor_api.cpp]: It'll create a walking section.

**Comment by xlqian** [comment]
ok for me!

**Pull request merged**